test/system/issues_test.rb

```diff
@@ -204,4 +204,53 @@ def test_issues_pagination
       assert_no_content(/extra_#{n}[^\d]/i)
     end
   end
+
+  def test_single_issue_reporters
+    sign_in_as(create(:moderator_user))
+    issue = create(:issue, :assigned_role => "moderator")
+    issue.reports << create(:report, :user => create(:user, :display_name => "Test Name"))
+
+    visit issues_path
+    assert_content issue.reported_user.display_name
+    assert_content issue.reports.first.user.display_name
+  end
+
+  def test_multiple_issue_reporters
+    sign_in_as(create(:moderator_user))
+    issue = create(:issue, :assigned_role => "moderator")
+
+    1.upto(5).each do |n|
+      issue.reports << create(:report, :user => create(:user, :display_name => "Test Name #{n}"))
+    end
```

Comment:
```suggestion
create_list(:report, 5, :issue => issue)
```
because you're not checking if usernames match `"Test Name #{n}"`.

And probably the same in the next test.

Reply:
Thanks for the suggestion. Both tests were changed to match smaller list creation you suggested.